wallet/internal/db/block_pg.go

```diff
@@ -0,0 +1,32 @@
+package db
+
+import (
+	"database/sql"
+	"fmt"
+	"time"
+
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+)
+
+// buildPgBlock constructs a Block from the given PostgreSQL block
+// fields.
+func buildPgBlock(
+	height sql.NullInt32, hash []byte, timestamp sql.NullInt64,
+) (*Block, error) {
```

Comment:
nit: I guess this is undefined format [here](https://github.com/btcsuite/btcwallet/blob/master/docs/developer/code_formatting_rules.md#wrapping-long-function-definitions), maybe we should update the docs if we decide to allow/forbid this format. And normally we would do, 
```go
func buildPgBlock(height sql.NullInt32, hash []byte, 
	timestamp sql.NullInt64) (*Block, error) {
```
same for the rest